R/get_ses.R

```diff
@@ -6,11 +6,11 @@
 #'
 #'@details This is an internal function not intended to be called directly.
 #'
-#'@seealso \code{\link{Ostats}} to calculate O-statistics (community-level
-#'  pairwise niche overlap statistics)
+#'@seealso \code{\link{Ostats}} to calculate O-statistics (pairwise niche overlap
```

Comment:
See above comment, should keep this as is

Reply:
added community-level back in